packages/components/src/custom-select-control-v2/legacy-component/test/index.tsx

```diff
@@ -347,14 +355,56 @@ describe.each( [
 			2,
 			expect.objectContaining( {
 				selectedItem: expect.objectContaining( {
-					// TODO: key should be different — this is a known bug and will be fixed
-					key: 'poppy',
+					key: 'flower3',
 					name: 'poppy',
 				} ),
 			} )
 		);
 	} );
 
+	it( 'Should accept and pass arbitrary properties to the selectedItem object in the onChange callback, but without applying them to the DOM elements apart from style and classname', async () => {
```

Comment:
How can we make this shorter 😅 Perhaps something like:

```suggestion
	it( "Should pass arbitrary props to onChange's selectedItem, but apply only style and className to DOM elements", async () => {
```